extensions/2.0/Vendor/GRIFFEL_bim_data/schema/node.GRIFFEL_bim_data.schema.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "GRIFFEL_bim_data Node Extension",
+    "type": "object",
+    "description": "References type and instance properties of the node.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "buffer": {
+            "type": "number",
+            "description": "Index of the buffer which contains data for this node. Used only when data is saved in a separate file."
+        },
+        "properties": {
+            "type": "array",
+            "items": {
+                "type": "number",
+                "description": "Index of a property in the root level collection."
+            },
+            "description": "Collection of indices which point to corresponding instance properties of the node. (Instance properties are unique to the node.)"
```

Comment:
Could per-instance property values override the values of the same name inherited from `type`?

Reply:
there is no difference, it can be overridden

Reply:
If per-node properties overwrite the same properties defined in the type, it must be stated here.